tools/apiview/parsers/rust-api-parser/rustdoc-types/LICENSE-APACHE.txt

```diff
@@ -0,0 +1,73 @@
+                                 Apache License
```

Comment:
Why Apache? We use MIT. If this is just a copy of rustc's, that needs to go into a ThirdPartyNotices.txt file.